packages/erebos-api-bzz-base/src/index.js

```diff
@@ -77,4 +77,16 @@ export default class BaseBzz {
   downloadRawText(hash: string): Promise<string> {
     return this.downloadRaw(hash).then(res => res.text())
   }
+
+  listDirectory(hash: string): Promise<string> {
```

Comment:
Shouldn't this request return a JSON object rather than a string?

Reply:
You're right, I see now that we did it differently for the functions of the same type. Fixed.